docs/Project.toml

```diff
@@ -6,6 +6,7 @@ Documenter = "0.24.2"
 
 [extras]
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
```

Comment:
the documentation Project.toml doesn't need to be edited.

Reply:
Ok, I'll change it.